src/main/java/ch/heigvd/pro/b04/sessions/Session.java

```diff
@@ -4,40 +4,74 @@
 import java.sql.Timestamp;
 import java.util.Random;
 import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.EmbeddedId;
 import javax.persistence.Entity;
 import javax.persistence.OneToMany;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.EqualsAndHashCode.Exclude;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 
-@Entity
 @NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@EqualsAndHashCode
 public class Session {
+  private static final int CODE_BOUNDARY = 0xFFFF;
```

Comment:
According to the [documentation](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt-int-), the parameter of `Random.nextInt(something)` is exclusive.